packages/utils/tests/assertion.test.ts

```diff
@@ -148,6 +148,7 @@ describe("Assertion", () => {
     test("Returns true for strings containing units", () => {
       expect(isUnit("100px")).toBeTruthy()
       expect(isUnit("5rem")).toBeTruthy()
+      expect(isUnit("2xl")).toBeFalsy()
     })
```

Comment:
I wonder if adding test code with `toBeFalsy()` to this test may cause a contradiction to its name: 

> Returns **true** for strings containing units

Maybe creating a new test case with a name like `Returns false for strings containing invalid units` is better.